general/development/process/translation/faq.md

```diff
@@ -15,7 +15,7 @@ import TabItem from '@theme/TabItem';
 
 ## How can I help with translating Moodle?
 
-Please see the guide [Contributing a translation](/general/development/process/translation/contributing).
+Please see the guide [Contributing a translation](./contributing.md).
```

Comment:
There is one more reference to /general/development/process/translation/contributing in line 22.